scripts/konflux/fail-build-if-git-is-dirty.sh

```diff
@@ -26,7 +26,7 @@ echo "If this command fails, you should see the list of modified files below."
 echo "You need to find the reason and prevent it because otherwise the build results will be inconsistent."
 echo ""
 
-if git status --porcelain | grep '.' >&2 ; then
+if git status --porcelain | grep -v "^??" | grep '.' >&2 ; then
```

Comment:
That's right, our `make tag` does not append `-dirty` if there are only untracked files. I wonder thought if we should allow them here or add to `.gitignore`. I haven't formed an opinion on this.

Reply:
what untracked files did you find? (I assume it's the downloaded ones?)

Reply:
> it would be good to put it in collector

Collector doesn't embed the version and doesn't have the `fail-build-if-git-is-dirty.sh` script, therefore noop.

Reply:
Unresolved because I have an alternative suggestion: instead of excluding untracked files, I suggest creating entries in `.gitignore` for these blobs. This will be more targeted than just ignoring any untracked files.

Not that I see a particular case where extra untracked files will cause harm but it feels cleaner. I'm also not sure how annoying it would be to keep failing on untracked files, it needs some time.